test/RBdata.sql

```diff
@@ -1996,7 +1996,7 @@ UNLOCK TABLES;
+INSERT INTO `media` (id, session_id, instrument, date_taken, comments, file_name, file_type, data_dir, uploaded_by, hide_file, date_uploaded) VALUES (1,2,NULL,'1001-01-01',NULL,'MTL002_V1.py','text/x-python-script','/data/uploads/','admin',0,'2016-08-11 14:10:57'),(2,2,NULL,'2016-12-31',NULL,'MTL002_V1.txt','text/plain','/data/uploads/','admin',0,'2016-08-11 14:06:43'),(3,181,NULL,NULL,NULL,'OTT181_V1.txt','text/plain','/data/uploads/','admin',0,'2016-08-15 14:21:15'),(4,1216,NULL,NULL,NULL,'OTT181_V2.txt','text/plain','/data/uploads/','admin',0,'2016-08-15 14:24:34'),(5,10,NULL,NULL,NULL,'MTL010_V1.txt','text/plain','/data/uploads/','admin',0,'2016-08-16 17:04:07'),(6,13,NULL,NULL,NULL,'MTL013_V1.txt','text/plain','/data/uploads/','admin',0,'2016-08-16 17:04:33'),(7,18,NULL,NULL,NULL,'MTL018_V1.txt','text/plain','/data/uploads/','admin',0,'2016-08-16 17:05:08'),(8,20,NULL,NULL,NULL,'MTL020_V1.txt','text/plain','/data/uploads/','admin',0,'2016-08-16 17:05:28'),(9,30,NULL,NULL,NULL,'MTL030_V1.txt','text/plain','/data/uploads/','admin',0,'2016-08-16 17:06:15'),(10,31,NULL,NULL,NULL,'MTL031_V1.txt','text/plain','/data/uploads/','admin',0,'2016-08-16 17:06:32'),(11,48,NULL,NULL,NULL,'MTL048_V1.txt','text/plain','/data/uploads/','admin',0,'2016-08-16 17:06:46'),(12,61,NULL,NULL,NULL,'MTL061_V1.txt','text/plain','/data/uploads/','admin',0,'2016-08-16 17:07:08'),(13,68,NULL,NULL,NULL,'MTL068_V1.txt','text/plain','/data/uploads/','admin',0,'2016-08-16 17:07:25'),(14,95,NULL,NULL,NULL,'MTL095_V1.txt','text/plain','/data/uploads/','admin',0,'2016-08-16 17:07:46'),(15,1079,NULL,NULL,NULL,'OTT187_V3.txt','text/plain','/data/uploads/','admin',0,'2016-08-16 17:08:08'),(16,1912,NULL,NULL,NULL,'OTT188_V2.txt','text/plain','/data/uploads/','admin',0,'2016-08-16 17:08:49'),(17,192,'bmi',NULL,NULL,'OTT192_V1_bmi.txt','text/plain','/data/uploads/','admin',0,'2016-08-16 17:09:13'),(18,260,NULL,'2016-05-05',NULL,'OTT260_V1.txt','text/plain','/data/uploads/','admin',0,'2016-08-16 17:09:47'),(19,285,'bmi',NULL,NULL,'OTT285_V1_bmi.txt','text/plain','/data/uploads/','admin',0,'2016-08-16 17:10:24'),(20,311,'bmi','2016-06-05',NULL,'OTT311_V1_bmi.txt','text/plain','/data/uploads/','admin',0,'2016-08-16 17:11:11'),(21,323,NULL,NULL,NULL,'OTT323_V1.txt','text/plain','/data/uploads/','admin',0,'2016-08-16 17:11:34'),(22,324,NULL,NULL,NULL,'OTT324_V1.txt','text/plain','/data/uploads/','admin',0,'2016-08-16 17:11:50'),(23,1359,NULL,NULL,NULL,'ROM474_V2.txt','text/plain','/data/uploads/','admin',0,'2016-08-16 17:12:33'),(24,854,NULL,NULL,NULL,'ROM474_V3.txt','text/plain','/data/uploads/','admin',0,'2016-08-16 17:12:50'),(25,862,NULL,NULL,NULL,'ROM482_V6.txt','text/plain','/data/uploads/','admin',0,'2016-08-16 17:13:13'),(26,1257,'medical_history',NULL,NULL,'ROM489_V3_medical_history.txt','text/plain','/data/uploads/','admin',0,'2016-08-16 17:13:31');
```

Comment:
> (id, session_id, instrument, date_taken, comments, file_name, file_type, data_dir, uploaded_by, hide_file, date_uploaded)

@kongtiaowang if we do this for all tables in RBdata this problem would not happen anymore